src/AuthContext.jsx

```diff
@@ -8,9 +9,24 @@ export const AuthProvider = ({ children }) => {
   const [isLoggedIn, setIsLoggedIn] = useState(false);
   const [userName, setUserName] = useState('');
 
+
+  useEffect(() => {
+    // Initialize state from localStorage
+    const storedIsLoggedIn = localStorage.getItem('isLoggedIn') === 'true';
+    const storedUserName = localStorage.getItem('userName') || '';
+
+    setIsLoggedIn(storedIsLoggedIn);
+    setUserName(storedUserName);
+  }, []);
+
   const handleLogin = (name) => {
     setIsLoggedIn(true);
     setUserName(name);
+
+    // Store state in localStorage
+    localStorage.setItem('isLoggedIn', 'true');
+    localStorage.setItem('userName', name);
+
   };
 
   const handleLogout = () => {
```

Comment:
In `handleLogin` we store state in `localStorage` with `isLoggedIn` and `userName`, but in `handleLogout` we are removing `authToken` and `userId`. Would it work?

Reply:
And what about export of AuthContext?